homework/js-exercises/Time/showCurrentTime.js

```diff
@@ -0,0 +1,26 @@
+const updateTime = () => {
+    if (document.querySelector('h1') !== null) {
+        lastTime = document.querySelector('h1');
+        lastTime.remove();
+    }
+
+    let date = new Date;
+    let minutes, seconds;
+
+    let time = document.createElement('h1');
+    if (date.getMinutes() < 10) {
+        minutes = `0${date.getMinutes()}`;
+    } else {
+        minutes = date.getMinutes();
+    }
+
+    if (date.getSeconds() < 10) {
+        seconds = `0${date.getSeconds()}`
+    } else {
+        seconds = date.getSeconds();
+    }
```

Comment:
This if seams to repeat the same logic for minutes and the seconds. Try to extract it into a function